full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -66,13 +86,455 @@ public void testWithNullUrlDir() {
+    public void testAddListenerWithApocLoadDirectoryNotEnabled() {
+        apocConfig().setProperty(APOC_IMPORT_FILE_USE_NEO4J_CONFIG, true);
+        apocConfig().setProperty(APOC_LOAD_DIRECTORY_ENABLED, false);
+        try {
+            db.executeTransactionally("CALL apoc.load.directory.add('test','CREATE (n:Test)', '*.csv', '', {}) YIELD name RETURN name");
+        } catch (RuntimeException e) {
+            assertEquals("Failed to invoke procedure `apoc.load.directory.add`: Caused by: java.lang.RuntimeException: " + LoadDirectory.NOT_ENABLED_ERROR, e.getMessage());
+            throw e;
+        }
+    }
+
+    @Test
+    public void testAddMultipleFoldersAndRemoveAll() throws InterruptedException, IOException {
+        apocConfig().setProperty(APOC_IMPORT_FILE_USE_NEO4J_CONFIG, false);
+        apocConfig().setProperty(APOC_LOAD_DIRECTORY_ENABLED, true);
+
+        final String importWithSlashEscaped = IMPORT_DIR.replace("\\", "\\\\");
+        File subTempFolder = new File(temporaryFolder.getRoot() + File.separator + importWithSlashEscaped + File.separator + SUBFOLDER_1);
+        String subTempFolderUrl = FILE_PROTOCOL + subTempFolder;
+        File subTempFolderTwo = new File(temporaryFolder.getRoot() + File.separator + importWithSlashEscaped + File.separator + SUBFOLDER_2);
+        String subTempFolderUrlTwo = FILE_PROTOCOL + subTempFolderTwo;
+        File subTempFolder3 = new File(temporaryFolder.getRoot() + File.separator + importWithSlashEscaped + File.separator + SUBFOLDER_3);
+        String subTempFolderUrl3 = FILE_PROTOCOL + subTempFolder3;
+
+        final Map<String, Object> defaultConfig = Map.of("eventKinds", DEFAULT_EVENT_KINDS, "interval", 1000L);
+        testCall(db, "CALL apoc.load.directory.add('testOne','CREATE (n:TestOne)','*.csv','" + subTempFolderUrl + "', {})", result -> {
+            assertEquals("testOne", result.get("name"));
+            assertEquals("*.csv", result.get("pattern"));
+            assertEquals(subTempFolderUrl.replace("\\\\", "\\"), result.get("urlDir"));
+            assertEquals("CREATE (n:TestOne)", result.get("cypher"));
+            assertEquals(defaultConfig, result.get("config"));
+            assertEquals(LoadDirectoryItem.Status.CREATED.name(), result.get("status"));
+            assertEquals(StringUtils.EMPTY, result.get("error"));
+        });
+
+        db.executeTransactionally("CALL apoc.load.directory.add('testTwo','CREATE (n:TestTwo)','*.json','" + subTempFolderUrlTwo + "', {}) YIELD name RETURN name");
+
+        testResult(db, "CALL apoc.load.directory.add('test3','CREATE (n:Test3)','*.html','" + subTempFolderUrl3 + "', {}) YIELD name RETURN name", result -> {
+            List<String> rows = Iterators.asList(result.columnAs("name"));
+            assertThat(rows, Matchers.containsInAnyOrder("testOne", "testTwo", "test3"));
+        });
+
+        final String queryCount = "MATCH (n:TestOne), (m:TestTwo), (l:Test3) RETURN count(n)+count(m)+count(l) AS count";
```

Comment:
Ma se fai 
```suggestion
        final String queryCount = "MATCH (n:TestOne), (m:TestTwo), (l:Test3) RETURN count(*) AS count";
```
non è lo stesso?